ui/pages/swaps/prepare-swap-page/prepare-swap-page.js

```diff
@@ -893,6 +893,7 @@ export default function PrepareSwapPage({
             display={DISPLAY.FLEX}
             justifyContent={JustifyContent.spaceBetween}
             alignItems={AlignItems.center}
+            gap={4}
```

Comment:
Adds gap between asset selector and input

<img width="420" alt="Screenshot 2024-12-17 at 8 49 40 AM" src="https://github.com/user-attachments/assets/f0c08ae2-1cd4-4eb8-807a-d8800a3d77c8" />